gpkit/repr_conventions.py

```diff
@@ -14,3 +15,16 @@ def _str(self):
 def _repr_latex_(self):
     "Returns default latex for automatic iPython Notebook rendering."
     return "$$"+self.latex()+"$$"
+
+
+def unitstr(units, into="%s", options="~", dimless=""):
+    "Returns the string corresponding to an object's units."
+    if hasattr(units, "units") and isinstance(units.units, Quantity):
+        units = units.units
+    if not isinstance(units, Quantity):
+        return dimless
+    rawstr = (u"{:%s}" % options).format(units.units)
+    if str(units.units) == "count":
```

Comment:
Oh, it's a pint version thing: we make sure that units is a Quantity, which means units.units is a Unit, and the str representation of Unit("count") is stable across pint versions while the str representation of Quantity("count") is not...